packages/docs/content/06-integrations/00-ubuntu.md

```diff
@@ -0,0 +1,28 @@
+---
+title: 'Ubuntu'
+metaTitle: 'Ubuntu Operating System'
+metaDescription: 'Choosing Ubuntu as an Operating System within the Spectro Cloud console'
+hiddenFromNav: true
+isIntegration: true
+category: ['operating system']
+logoUrl: 'https://registry.spectrocloud.com/v1/ubuntu-vsphere/blobs/sha256:09a727f9005b79c69d8e60e12ce130880c63131315b49e7fb4cc44e53d34dc7a?type=image/png'
+---
+
+import WarningBox from '@librarium/shared/src/components/WarningBox';
+
+
+
+# Ubuntu Operating System
+
+## Overview
+
+Ubuntu is a Linux distribution based on Debian and composed mostly of free and open-source softwares. It is one of the most popular operating systems across multiple public cloud platforms. Spectro Cloud supports Ubuntu versions to run clusters at scale. 
+
+## Versions supported:

```

Comment:
Can we have a common layout for both OS. Versions here version there . References here ##References can . can be uniform

Reply:
sure